Tests/ImagineEngineTests/Mocks/TextureErrorHandlerMock.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  TextureErrorHandlerMock.swift
+//  ImagineEngine
+//
+//  Created by Vijay on 21/11/17.
+//  Copyright © 2017 ImagineEngine. All rights reserved.
+//
+
+import Foundation
+@testable import ImagineEngine
+
+class TextureErrorHandlerMock: TextureErrorHandler {
+
```

Comment:
🎨 No new lines at the beginning of a type definition (should add a SwiftLint rule for that)

Reply:
Fixed 👍 
Will check about the SwiftLint rule.